packages/invoices/src/OutboundShipment/DetailView/reducer.test.ts

```diff
@@ -20,6 +21,11 @@ describe('DetailView reducer', () => {
       costPricePerPack: 0,
       sellPricePerPack: 0,
       itemName: 'a',
+      stockLineId: '',
+      invoiceId: '',
+      isCreated: false,
+      isUpdated: false,
+      isDeleted: false,
```

Comment:
I've added these `isX` fields to track what mutation to perform for each row. I've used them from the perspective of the client.. as in "isCreated" is `true` when it is created on the client but not persisted on the server, `isDeleted` is true when the row is deleted on the client, but not on the server. I was unsure whether it should be the reverse - i.e. `isCreated` is `true` when it is a persisted row, and when we create one on the client we set that to `false`  😕